core/region.go

```diff
@@ -303,9 +303,7 @@ func (r *region) scanForEnabledAutoScalingGroups() {
 
 	svc := r.services.autoScaling
 
-	input := autoscaling.DescribeAutoScalingGroupsInput{
-		AutoScalingGroupNames: asgNames,
-	}
+	input := autoscaling.DescribeAutoScalingGroupsInput{}
```

Comment:
You can inline the `autoscaling.DescribeAutoScalingGroupsInput{}` below and delete this line entirely.